README.md

```diff
@@ -599,7 +599,7 @@ Options:
   -f, --defintionFile <path_or_url>      The definition file, either a path to the filesystem or a URL to it
   -o, --outputFolder <path>              The folder path to store projects. Default is of the format 'build_chain_yyyymmddHHMMss' (default:
                                          "build_chain_202211229567")
-  --token <token>                        The GITHUB_TOKEN. It can be set as an environment variable instead
+  --token <token...>                     The GITHUB_TOKEN. It can be set as an environment variable instead. You can specify multiple tokens
```

Comment:
How to I pass multiple tokens ? comma separated ? using multiple times the `--token` option ?
Does it work also with the env variable ?

Reply:
this is about to execute `--token token1 token2...`
about env, it is not possible according to current proposal (as far as I can see), right Shub?

Reply:
Yep not possible through env cause I thought I will keep GITHUB_TOKEN usage consistent and aligned with what users already think it is